draft-ietf-quic-transport.md

```diff
@@ -2377,18 +2377,18 @@ A connection that remains idle for longer than the advertised idle timeout (see
 {{transport-parameter-definitions}}) is closed.  A connection enters the
 draining state when the idle timeout expires.
 
-Endpoints use the value they advertise when determining an idle timeout.  The
-idle timeout starts from the last packet received, or the first packet sent
-after sending that packet.  The latter condition ensures that initiating new
-activity postpones a timeout.
-
-The value for an idle timeout can be asymmetric.  The value advertised by a peer
-is only used to determine whether the connection is live at a peer.  An endpoint
-that sends packets near the end of the idle timeout period of a peer risks
-having those packets discarded if its peer enters the draining state before the
-packets arrive.  If a peer could timeout within an RTO (see Section 4.3.3 of
-{{QUIC-RECOVERY}}), it is advisable to test for liveness before sending any data
-that cannot be retried safely.
+Each endpoint advertises their own idle timeout to their peer. The idle timeout
+starts from the last packet received, or the first packet sent after sending
+that packet.  The latter condition ensures that initiating new activity
+postpones a timeout.
```

Comment:
This is not very clear without more context:
What is "that" packet? Do you mean "The idle timeout is reset whenever a packet is received or sent."?
 Where is the timeout advertised, TP presumably?

Reply:
OK there is a reference to TP in earlier paragraph.

Reply:
@mikkelfj, you picked up on a serious error here.  It's the first packet sent after the last received packet.  Fresh eyes FTW.